buildkite/aggregate_incompatible_flags_test_result.py

```diff
@@ -264,16 +284,33 @@ def print_flags_need_to_migrate(failed_jobs_per_flag, details_per_flag):
         if jobs:
             github_url = details_per_flag[flag].issue_url
             info_text = [f"* **{flag}** " + get_html_link_text(":github:", github_url)]
-            info_text += merge_and_format_jobs(jobs.values(), "  - **{}**: {}")
+            jobs_per_pipeline = merge_jobs(jobs.values())
+            for pipeline, platforms in jobs_per_pipeline.items():
+                color = "red" if bazelci.DOWNSTREAM_PROJECTS[pipeline].get("migration_owner") == "Bazel Team" else "black"
+                platforms_text = ", ".join(platforms)
+                info_text.append(f"  <li><strong style=\"color: {color}\">{pipeline}</strong>: {platforms_text}</li>")
             # Use flag as the context so that each flag gets a different info box.
             print_info(flag, "error", info_text)
             printed_flag_boxes = True
     if not printed_flag_boxes:
         return
-    info_text = ["#### Downstream projects need to migrate for the following flags:"]
+    info_text = [
+        "#### Downstream projects need to migrate for the following flags:",
+        "    Projects with <strong style=\"color: red;\">red title</strong> need to be migrated by Bazel team.",
+    ]
     print_info("flags_need_to_migrate", "error", info_text)
 
 
+def merge_jobs(jobs):
+    jobs = list(jobs)
+    jobs.sort(key=lambda s: s["name"].lower())
```

Comment:
These two lines can be merged with line 308:

for job in sorted(jobs, key=...):

It's short enough to

Reply:
Thanks!